Marlin/Marlin_main.cpp

```diff
@@ -7040,9 +7064,11 @@ void process_next_command() {
           break;
       #endif // ENABLED(FILAMENT_WIDTH_SENSOR)
 
-      case 410: // M410 quickstop - Abort all the planned moves.
-        gcode_M410();
-        break;
+      #if DISABLED(EMERGENCY_PARSER)
+        case 410: // M410 quickstop - Abort all the planned moves.
+          gcode_M410();
+          break;
+      #endif
```

Comment:
All it needs to not trow "unknown command" is A 'G', 'M' or 'T' followed by a number. If the number is not handled it does not matter. It is this way to not throw errors on unknown command numbers by purpose.